File: server/router/business/mem_user.go
package business

import (
	"go-cms/api/v1"
	"go-cms/middleware"
	"github.com/gin-gonic/gin"
)

type MemUserRouter struct {
}

// InitMemUserRouter 初始化 MemUser 路由信息
func (s *MemUserRouter) InitMemUserRouter(Router *gin.RouterGroup) {
	router := Router.Group("memUser").Use(middleware.OperationRecord())
	routerNoRecord := Router.Group("memUser")
	var apiV1 = v1.ApiGroupApp.BusinessApiGroup.MemUserApi
	{
		router.POST("createMemUser", apiV1.CreateMemUser)   // 新建MemUser
		router.DELETE("deleteMemUser", apiV1.DeleteMemUser) // 删除MemUser
		router.DELETE("deleteMemUserByIds", apiV1.DeleteMemUserByIds) // 批量删除MemUser
		router.PUT("updateMemUser", apiV1.UpdateMemUser)    // 更新MemUser
	    router.POST("quickEdit", apiV1.QuickEdit)  // 快速编辑	
	}
	{
		routerNoRecord.GET("findMemUser", apiV1.FindMemUser)        // 根据ID获取MemUser
		routerNoRecord.GET("getMemUserList", apiV1.GetMemUserList)  // 获取MemUser列表
        routerNoRecord.GET("excelList", apiV1.ExcelList)  // 分页导出excel MemUser列表
	}
}

File: web/src/api/memUser.ts
import { http,resData } from "@/utils/http";

// @Tags MemUser
// @Summary 创建MemUser
export const createMemUser = (data?: object) => {
   return http.request<resData>("post", "/memUser/createMemUser", { data });
}

// @Tags MemUser
// @Summary 删除MemUser
export const deleteMemUser = (data?: object) => {
   return http.request<resData>("post", "/memUser/deleteMemUser", { data });
}


// @Tags MemUser
// @Summary 删除ByIdsMemUser
export const deleteMemUserByIds = (data?: object) => {
   return http.request<resData>("post", "/memUser/deleteMemUserByIds", { data });
}


// @Tags MemUser
// @Summary 更新MemUser
export const updateMemUser = (data?: object) => {
   return http.request<resData>("post", "/memUser/updateMemUser", { data });
}

// @Tags MemUser
// @Summary 快速编辑
export const quickEdit = (data?: object) => {
   return http.request<resData>("post", "/memUser/quickEdit", { data });
}


// @Tags MemUser
// @Summary 用id查询MemUser
export const findMemUser = (params?: object) => {
   return http.request<resData>("get", "/memUser/findMemUser", { params });
}


// @Tags MemUser
// @Summary 分页获取MemUser列表
export const getMemUserList = (params?: object) => {
   return http.request<resData>("get", "/memUser/getMemUserList", { params });
}


// @Tags MemUser
// @Summary 导出excelMemUser列表
export const excelList = (params?: object) => {
   return http.request<resData>("get", "/memUser/excelList", { params });
}


// @Tags jqCustomerPwd
// @Summary 修改密码
export const memChangePwd = (data?: object) => {
   return http.request<resData>("post", "/memUser/memChangePwd", { data });
}


